{{cookiecutter.project_slug}}/README.rst

```diff
@@ -64,7 +64,11 @@ Initial Setup (APPUiO + GitLab)
 
    -  Operations > Kubernetes > "APPUiO" > Kubernetes cluster details > Service Token
 
-   (*Note:* Make sure "GitLab-managed cluster" is unchecked in the cluster details.)
+   and ensure the following values are set in the cluster details:
+
+   - RBAC-enabled cluster: *(unchecked)*
```

Comment:
```suggestion
   - RBAC-enabled cluster: *(checked)*
```

Even though it technically doesn't matter since `GitLab-managed` is unchecked, the cluster is RBAC-enabled...

Reply:
Thanks! I applied the change to the original commit.